### PR TITLE
Updated the project

### PR DESCRIPTION
**Pull request by GordonOtieno**
- Created cars controllers methods
- cars routes
- solved linters errors

### CODE REVIEW

File: app/controllers/api/v1/cars_controller.rb [hunk 1]
```diff
@@ -1,12 +1,40 @@
 class Api::V1::CarsController < ApplicationController
   def index
     @cars = Car.all
-    render json: @cars
+    if @cars.empty?
+      render json: { message: 'No cars found' }
+    else
+      render json: @cars
+    end
```
Comment (ouasamine) [inline app/controllers/api/v1/cars_controller.rb]: You could add responses status codes here, to make API consumption easy and straightforward.

File: app/controllers/api/v1/cars_controller.rb [hunk 2]
```diff
@@ -1,12 +1,40 @@
...
   end
 
-  def new; end
+  def show
+    @car = Car.find(params[:id])
+    render json: @car
+  end
```
Comment (ouasamine) [inline app/controllers/api/v1/cars_controller.rb]: Status codes could be used here as well and if statement as well to handle cases where the car is not found.

**Review by ouasamine — APPROVED** [review]
Hi @GordonOtieno,
Excellent work on this controller, I have added some recommendations below, please make sure to implement them before merging.
Other than that this PR is good to be merged ✔️

File: app/controllers/api/v1/cars_controller.rb (hunk 2 above)
Comment (GordonOtieno) [reply app/controllers/api/v1/cars_controller.rb]: alrigh. let me do

**Pull request merged**